chart/githubconnector/README.md

```diff
@@ -7,7 +7,7 @@
 	- [Steps](#steps)
 
 ## Overview
-The chart creates a GitHub Connector Deployment, and a Namespace inside Kyma.
+The chart creates a GitHub Connector Deployment and a Namespace inside Kyma.
```

Comment:
Sorry I missed that, but Kyma is not a container so you don't really say that you create things inside of it. Say "in Kyma" instead ;)